packages/form-state-manager/src/types/validate.d.ts

```diff
@@ -1,9 +1,13 @@
 import AnyObject from './any-object';
 import { ManagerApi } from './manager-api';
 
+export interface FormLevelError {
+  [key: string]: string;
```

Comment:
I think we should support more types. I remember I was using `<FormattedMessage />` to return translated message in Sources and it worked.

Reply:
Yeah, I was thinking about this as well. I think we might have to re-visit this topic because the field level errors are locked to strings as well. And also all validator return values.